gm4_tnt_landmines/data/tnt_landmines/functions/disarming.mcfunction

```diff
@@ -0,0 +1,22 @@
+execute if entity @e[type=armor_stand,tag=gm4_tnt_mine,distance=..0.1,nbt={ArmorItems:[{},{},{},{id:"minecraft:grass_block",Count:1b}]}] run summon item ~ ~1 ~ {Motion:[0.0,0.4,0.0],Item:{id:":player_head",Count:1b,tag:{SkullOwner:{Id:3810590c-6484-4133-b456-a3fc378a39a3,Properties:{textures:[{Value:"eyJ0aW1lc3RhbXAiOjE1NTQ0NzQwMDkwMDUsInByb2ZpbGVJZCI6IjNmYzdmZGY5Mzk2MzRjNDE5MTE5OWJhM2Y3Y2MzZmVkIiwicHJvZmlsZU5hbWUiOiJZZWxlaGEiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzZiMzBmZTgxYWZlNTI2NjkxZGM1MDlhYzEwN2EzYjNjYTk1NTdjNDExZmYwZGMyZmNiNWE4YTlkNzc1ZTk0ODMifX19"}]}},gm4_tnt_landmines:{item:1,block:"grass"},display:{Name:"{\"text\":\"Landmine\"}",Lore:["{\"text\":\"Drop to use\"}","{\"text\":\"Disguised as a grass block\"}"]}}}}
+execute if entity @e[type=armor_stand,tag=gm4_tnt_mine,distance=..0.1,nbt={ArmorItems:[{},{},{},{id:"minecraft:dirt",Count:1b}]}] run summon item ~ ~1 ~ {Motion:[0.0,0.4,0.0],Item:{id:":player_head",Count:1b,tag:{SkullOwner:{Id:da309950-dcbc-43f2-880e-3820cffea8ca,Properties:{textures:[{Value:"eyJ0aW1lc3RhbXAiOjE1NTQ0NzQ0MjA4NDgsInByb2ZpbGVJZCI6IjkxZjA0ZmU5MGYzNjQzYjU4ZjIwZTMzNzVmODZkMzllIiwicHJvZmlsZU5hbWUiOiJTdG9ybVN0b3JteSIsInNpZ25hdHVyZVJlcXVpcmVkIjp0cnVlLCJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvZGI1Mzc5ZjE2NWE1Y2M3YmI4YzM4MmZlOTJlMjRlODYwN2NiODU1Y2FjNmU4M2Q4NjJjNjBlZmNlZTA5MTE3NSJ9fX0="}]}},gm4_tnt_landmines:{item:1,block:"dirt"},display:{Name:"{\"text\":\"Landmine\"}",Lore:["{\"text\":\"Drop to use\"}","{\"text\":\"Disguised as dirt\"}"]}}}}
+execute if entity @e[type=armor_stand,tag=gm4_tnt_mine,distance=..0.1,nbt={ArmorItems:[{},{},{},{id:"minecraft:stone",Count:1b}]}] run summon item ~ ~1 ~ {Motion:[0.0,0.4,0.0],Item:{id:":player_head",Count:1b,tag:{SkullOwner:{Id:ec0d8396-3a18-4298-842c-de978a5ee5c6,Properties:{textures:[{Value:"eyJ0aW1lc3RhbXAiOjE1NTQ0NzQxODg2MTMsInByb2ZpbGVJZCI6IjNmYzdmZGY5Mzk2MzRjNDE5MTE5OWJhM2Y3Y2MzZmVkIiwicHJvZmlsZU5hbWUiOiJZZWxlaGEiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzNjNjk4ZTE5MDllMWI4MzMxYmVhOGQ1YzU3N2NkMGM2M2I4YWVjMDFmYjg4ZjY0OWUwMGYwZmU5NjY1NDdjNWIifX19"}]}},gm4_tnt_landmines:{item:1,block:"stone"},display:{Name:"{\"text\":\"Landmine\"}",Lore:["{\"text\":\"Drop to use\"}","{\"text\":\"Disguised as stone\"}"]}}}}
```

Comment:
You do the translations like this:
```
Lore:[
    '{"translate":"%1$s%3427655$s","with":["Drop to use",{"translate":"item.gm4.tnt_landmine.use"}]}',
    '{"translate":"%1$s%3427655$s","with":["Disguised as stone",{"translate":"item.gm4.tnt_landmine.disguised.stone"}]}'
]
```

Reply:
Thank you was not sure how this worked, i have added it now